enforcer-rules/src/it/bom-project-using-spring-repository/verify.groovy

```
@@ -12,4 +12,4 @@ assert !buildLog.text.contains(
 
 assert !buildLog.text.contains("NullPointerException")
 
-assert buildLog.text.contains("Linkage Checker rule found 854 errors:")
\ No newline at end of file
+assert buildLog.text.contains("Linkage Checker rule found 850 errors:")
```

MethodHandle linkage errors are not reported.